tableaudocumentapi/column.py

```diff
@@ -0,0 +1,62 @@
+import functools
+
+_ATTRIBUTES = [
+    'name',
+    'caption',
+    'datatype',
+    'role',
+    'type',
+    'alias',
+    'calculation',
+]
+
+_METADATA_ATTRIBUTES = [
+    'aggregation',
+]
+
+
+def _find_metadata_record(record, attrib):
+    element = record.find('.//{}'.format(attrib))
+    if element is None:
+        return None
+    return element.text
+
+
+class Column(object):
```

Comment:
hrm, yeah I was basing it on what the code calls it.  If our literature calls it a Field, we should call it field.  They are basically the same thing.